src/app/app.component.scss

```diff
@@ -0,0 +1,12 @@
+.login{

```

Comment:
Why all this style is here?

If this style is common to all components then move it to styles.scss. And if the style is for individual component then move it to respective SCSS file.